terraform/datadog/tests/unit.tftest.hcl

```diff
@@ -7,11 +7,12 @@ variables {
   environment = "test-env"
   repos       = ["test-repo1", "test-repo2"]
   config = {
-    team_name         = "test-team",
-    contact_name      = "test-contact-name",
-    contact_email     = "test-contact-email",
-    documentation_url = "test-docs",
-    #services_to_monitor = ["test-web", "test-postgres"] #, "test-redis", "test-elasticsearch"]
+    team_name           = "test-team",
+    contact_name        = "test-contact-name",
+    contact_email       = "test-contact-email",
+    contacts            = yamlencode({ "email" : [{ "name" : "d", "address" : "a" }] })
+    documentation_url   = "test-docs",
+    description         = "test application",
     services_to_monitor = { "test-web" : ["postgres", "redis"], "test-api" = ["nginx", "ipfilter"] }
```

Comment:
Do we need to think about the consistency of using `:` vs `=` for object/map key, value pairs?